src/compiler.js

```diff
@@ -1103,7 +1103,7 @@ var Compiler = Object.extend({
 // console.log(tmpl);
 
 module.exports = {
-    compile: function(src, asyncFilters, extensions, name, lexerTags) {
+    compile: function(src, asyncFilters, extensions, name, lexerTags, trimBlocks, lstripBlocks) {
```

Comment:
Let's pass around a `options` object instead. If you can, move all the arguments after `src` into an `options` argument and us it like `opts.trimBlocks`. I think it would be fine to pass this same options argument to the parser and lexer as well.